### PR TITLE
Ransom Questions

### PR DESCRIPTION
**Pull request by DavidHaighton**

### CODE REVIEW

File: TheEthicalHackerCup/Assets/Resources/Ransomware1.xml [hunk 1]
```diff
@@ -1,20 +1,55 @@
...
+			<Selection>0</Selection>
 			<Correct>1</Correct>
-			<Option>Hello</Option>
-			<Option>Bruh</Option>
+			<Option>false</Option>
+			<Option>true</Option>
 		</Content>
 	</Slide>
-	<Slide Name="SLIDE 3">
+	<Slide Name="The hackers that made the ransomware will give you the password to your files if you ask nicely">
```
Comment (MillanWang) [inline TheEthicalHackerCup/Assets/Resources/Ransomware1.xml:47]: image
Since title text does not currently resize itself, some of this question will be cut off

File: TheEthicalHackerCup/Assets/Resources/Ransomware2.xml [hunk 2]
```diff
@@ -1,20 +1,39 @@
...
-			<Option>Bruh</Option>
+	<Slide Name="Avoidance">
+		<Content Type="Learning.InfoContent">Avoiding ransomware, and malware in general is simple from a users point of view. As long as their system stays up to date and the only things being downloaded are from trusted sources, this in most cases is suffient.</Content>
+	</Slide>
+	<Slide Name="Detection">
+		<Content Type="Learning.InfoContent">Antivirus is the second line of defense. If someone installs malware, antivirus is a type of program designed to detect and prevent the malware from running.</Content>
+	</Slide>
+	<Slide Name="Detection">
+		<Content Type="Learning.InfoContent">Antivirus used to classify files via signiture detection. This is where it would analyze the contents of the file and see if it recognizes any patterns of known viruses. Modern antiviruses work through heuristic detection alongside signiture detection. Heuristic detection is where the viruses are detected by running the program and analyzing its behaviour within an environment where the virus wont cause any damage.</Content>
+	</Slide>
```
Comment (MillanWang) [inline TheEthicalHackerCup/Assets/Resources/Ransomware2.xml:13]: image
This would fit better as 2 separate slides

File: TheEthicalHackerCup/Assets/Resources/Ransomware2.xml [hunk 3]
```diff
@@ -1,20 +1,39 @@
...
+	<Slide Name="Antivirus is the first line of defense against ransomware">
+		<Content Type="Learning.RadioBox">
+			<Selection>0</Selection>
+			<Correct>0</Correct>
+			<Option>False</Option>
+			<Option>True</Option>
 		</Content>
 	</Slide>
-	<Slide Name="SLIDE 3">
+	<Slide Name="Computer users should download anything from anywhere or plug anything into their computer without worry of ransomware">
```
Comment (MillanWang) [inline TheEthicalHackerCup/Assets/Resources/Ransomware2.xml:22]: image
Length

File: TheEthicalHackerCup/Assets/Resources/Ransomware3.xml [hunk 4]
```diff
@@ -1,20 +1,27 @@
...
+		<Content Type="Learning.InfoContent">Welcome to the third Ransomware Quiz. This quiz contains information and questions about the dealing with ransomware</Content>
 	</Slide>
-	<Slide Name="SLIDE 2">
-		<Content Type="Learning.CheckBox">
-			<Correct>1</Correct>
-			<Option>Hello</Option>
-			<Option>Bruh</Option>
-		</Content>
+	<Slide Name="Meeting the ransomers demands">
+		<Content Type="Learning.InfoContent">If your system is ever the victim of ransomware, it is best to not pay the ransomers demands. This is because it is never guaranteed that the ransomer will give you the key/password once you pay. Also, paying the ransomers gives ransomers means and insentive to continue.</Content>
```
Comment (MillanWang) [inline TheEthicalHackerCup/Assets/Resources/Ransomware3.xml:6]: incentive instead of insentive

File: TheEthicalHackerCup/Assets/Resources/Ransomware2.xml [hunk 5]
```diff
@@ -1,20 +1,39 @@
...
 		<Content Type="Learning.RadioBox">
-			<Selection>1</Selection>
+			<Selection>0</Selection>
+			<Correct>0</Correct>
+			<Option>False</Option>
+			<Option>True</Option>
+		</Content>
+	</Slide>
+	<Slide Name="Modern antiviruses work via">
```
Comment (MillanWang) [inline TheEthicalHackerCup/Assets/Resources/Ransomware2.xml:30]: This question only shows 2 options. 

If you move this question before the other True/False questions, then it shows all 4 options. Easiest fix at this point

File: TheEthicalHackerCup/Assets/Resources/Ransomware1.xml (hunk 1 above)
Comment (DavidHaighton) [reply TheEthicalHackerCup/Assets/Resources/Ransomware1.xml:47]: I think Jason said he fixed it for his quiz. I'm not going to touch it to avoid a merge conflict.

Comment (MillanWang) [reply TheEthicalHackerCup/Assets/Resources/Ransomware1.xml:47]: Even with Jason's fix, it goes an extra line down that's partially cut off. You only gotta reword it to make it slightly shorter. 

`"Ransomware makers will unlock your files if you ask nicely"` could work as that is easily 2-lineable

**Review by NosBos — APPROVED** [review]
